src/components/AdminSettings.vue

```diff
@@ -643,9 +691,16 @@ export default {
 						|| this.models[1]
 						|| this.models[0]
 
+					const defaultTtsModelId = this.state.default_stt_model_id || response.data?.default_stt_model_id
```

Comment:
I know it's a draft PR. Just in case you are wondering:
```suggestion
					const defaultTtsModelId = this.state.default_speech_model_id || response.data?.default_stt_model_id
```
and you need to adjust the `onModelSelected` method.
In the backend, the `default_stt_model_id` needs to be declared as a setting key in `lib/Service/OpenAiSettingsService.php` (add it in `ADMIN_CONFIG_TYPES` and implement getter+setter).